### ISSUE TITLE
Missing labels for WAV2VEC2_XLSR53

### ISSUE THREAD

**Issue by ntakouris** [pytorch/audio#3437]
### 🐛 Describe the bug

Trying to follow [character-level CTC segmentation for forced alignment with wav2vec2](https://pytorch.org/audio/stable/tutorials/forced_alignment_tutorial.html)

Usually, all the models contain some sort of label manifest, like this [like this](https://github.com/pytorch/audio/blob/c76d952e78728e9bf69c8c9c2a905209385f0f06/torchaudio/pipelines/_wav2vec2/impl.py#L752) 

The specific [XLSR53](https://github.com/pytorch/audio/blob/c76d952e78728e9bf69c8c9c2a905209385f0f06/torchaudio/pipelines/_wav2vec2/impl.py#LL770C1-L770C17) does not include a labels manifest, making it unusable for the CTC segmentation

### Versions

NA

**Comment by mthrok** [comment]
Hi 

WAV2VEC2_XLSR53 is a pre-trained on unlabeled data, but not it's not fine-tuned on any task, thus it does not have a label. (so it cannot be used for FA out-of-box)

If I remember correctly, they don't distribute the fine-tuned model weights.

**Comment by mthrok** [comment]
If you are looking to do multilingual FA, the model from this tutorial can be used.

https://pytorch.org/audio/main/tutorials/ctc_forced_alignment_api_tutorial.html

@nateanl Let's add the model bundle, this should be helpful to many people.
